src/all/mangadex/src/eu/kanade/tachiyomi/extension/all/mangadex/MangaDexHelper.kt

```diff
@@ -328,17 +326,7 @@ class MangaDexHelper() {
                 }
             }
 
-            if (!attr.externalUrl.isNullOrEmpty() && !attr.externalUrl.contains(BILIBILI_URL)) {
-                return null
-            }
-
-            // Bilibili special check. If it's a Bilibili chapter and the
-            // publishAt date is < now, it can be read on MD.
-            if (
-                !attr.externalUrl.isNullOrEmpty() &&
-                attr.externalUrl.contains(BILIBILI_URL) &&
-                parseDate(attr.publishAt) >= Date().time
-            ) {
+            if (!attr.externalUrl.isNullOrEmpty() && attr.pages == 0) {
```

Comment:
You shouldnt have to worry about empty strings, so it could be simplified to 
```suggestion
            if (attr.externalUrl != null && attr.pages == 0) {
```
Otherwise this looks good